share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -98,3 +108,38 @@ macro(rocm_create_package)
     endif()
     include(CPack)
 endmacro()
+
+
+function (rocm_set_system_name)
+    set(_system_name "unknown")
+    rocm_set_os_id(_os_id)
+    read_key("VERSION_ID" _version_id)
+
+    if (_os_id_centos OR _os_is_rhel)
+        STRING(CONCAT _system_name "el" ${_version_id} ".x86_64")
+    else()
+        STRING(CONCAT _system_name ${_os_id} "-"$ {_version_id} ".amd64")
+    endif()
+    rocm_set_parent(CMAKE_SYSTEM_NAME ${_system_name})
+endfunction()
+
+function (rocm_set_os_id OS_ID)
+    set(_os_id "unknown")
+    if (EXISTS "/etc/os-release")
+        rocm_read_os_release("ID" _os_id)
+        endif()
+    set(${OS_ID} ${_os_id} PARENT_SCOPE)
+    set(${OS_ID}_${_os_id} TRUE PARENT_SCOPE)
+endfunction()
+
+function (rocm_read_os_release KEYVALUE OUTPUT)
```

Comment:
Its common to have the `OUTPUT` variable first in cmake. Maybe this should have the `OUTPUT` variable first?